File: swarm-keydb-js/examples/chat-message-history.mjs
import { SwarmKeyDb } from '../index.js';

const db = new SwarmKeyDb({ host: '127.0.0.1', port: 6379 });
await db.connect();
await db.setWithTTL('chat:room1:last', JSON.stringify({ from: 'alice', text: 'hi' }), 60);
console.log(await db.list('chat:*'));
await db.disconnect();

File: swarm-keydb-py/examples/chat_message_history.py
import asyncio

from swarm_keydb import AsyncSwarmKeyDb


async def main():
    client = AsyncSwarmKeyDb(host="127.0.0.1", port=6379)
    await client.set_with_ttl("chat:room1:last", '{"from":"alice","text":"hi"}', 60)
    print(await client.list("chat:*"))
    await client.close()


asyncio.run(main())
